apps/cardio-patient/src/app/app.module.ts

```diff
@@ -55,13 +55,16 @@ function initializeFhirClientFactory(service: FhirDataService): () => Promise<vo
     FhirChartTagsLegendComponent,
   ],
   providers: [
-    { provide: APP_INITIALIZER, useFactory: initializeFhirClientFactory, deps: [FhirDataService], multi: true },
+    provideAppInitializer(() => {
+      const initializerFn = initializeFhirClientFactory(inject(FhirDataService));
+      return initializerFn();
```

Comment:
Can we can simplify it to
`provideAppInitializer(() => inject(FhirDataService).initialize())`